src/filmgrain.rs

```diff
@@ -387,26 +387,30 @@ unsafe fn generate_grain_y_rust<BD: BitDepth>(
     }
 
     let ar_pad = 3;
-    let ar_lag = data.ar_coeff_lag as isize;
+    // `ar_lag` is 2 bits; this tell the compile it definitely is.
```

Comment:
```suggestion
    // `ar_lag` is 2 bits; this tells the compile it definitely is.
```

Reply:
By the way, do you know how to use the "commit suggestion" button but also rebase/amend the previous commit?